element_lab/lab.py

```diff
@@ -32,24 +42,45 @@ class Lab(dj.Lookup):
 
 @schema
 class Location(dj.Lookup):
+    """Location of animal housing or experimental rigs.
+
+    Attributes:
+        Lab (foreign key): Lab key.
+        location ( varchar(32) ): Location of the lab.
+        location_description ( varchar(255) ): Optional. Description of the lab location.
+    """
+
     definition = """
-    # location of animal housing or experimental rigs
     -> Lab
-    location            : varchar(32)
+    location                   : varchar(32)
     ---
     location_description=''    : varchar(255)
     """
 
 
 @schema
 class UserRole(dj.Lookup):
+    """Roles assigned to a user or a job title.
+
+    Attributes:
+        user_role ( varchar(16) ): (e.g., "PI", "Postdoc", "Surgeon", etc.)
+    """
+
     definition = """
-    user_role       : varchar(16)
+    user_role           : varchar(16)
     """
 
 
 @schema
 class User(dj.Lookup):
+    """Table for storing user information.
+
+    Attributes:
+        user ( varchar(32) ): User name.
+        user_email ( varchar(128) ): User email address.
+        user_cellphone ( varchar(32) ): User cellphone number.
+    """
+
     definition = """
     user                : varchar(32)
```

Comment:
Please add comments to table definitions to match docstrings, excluding foreign key references